napari/_vispy/overlays/bounding_box.py

```diff
@@ -31,7 +31,6 @@ def _on_bounds_change(self):
             bounds = np.pad(bounds, ((1, 0), (0, 0)))
 
         self.node.set_bounds(bounds[::-1])  # invert for vispy
-        self._on_lines_change()
```

Comment:
This was no longer needed since the overlay no longer uses different visuals for 2D and 3D.